CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed talent page to show decoded capaiblities. [#703](https://github.com/rokwire/rokwire-building-blocks-api/issues/703)
 - Fix title search with user auth key. [#732](https://github.com/rokwire/rokwire-building-blocks-api/issues/732)
 
+### Fixed
+- Contribution POST gives proper error message. [#714](https://github.com/rokwire/rokwire-building-blocks-api/issues/714)
```

Comment:
Sorry, I did not see this early. This also needs to be moved to the unreleased section. Thank you.

Reply:
changed

Reply:
Thanks.